frontend/src/views/web/challenge/overview.html

```diff
@@ -25,5 +25,14 @@
                 <div ng-bind-html="challenge.page.description"></div>
             </div>
         </div>
+        <div ng-if="challenge.isChallengeHost" class="col s1">
+            <div class="row">
+                <div class="col s12 m12">
+                    <div class="align-right">
+                        <a class="waves-effect waves-dark btn ev-btn-dark w-300 fs-14" ng-click="challenge.deleteChallengeDialog($event)">Delete Challenge</a>
```

Comment:
I think `delete` should be next to the edit icon and it should be a `delete` icon in red color and when you hover on that then it will show `Delete Challenge`. Please let me know if you have doubts in it!